adobe/react-spectrum
# fix(#5745): Allow typing in Accordion

Closes #5745 

Set `disallowTypeAhead: true` for `Accordion` to allow typing in items. Add relevant story and test.

The [ARIA Accordion Pattern's keyboard interactions](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/#:~:text=at%20a%20time.-,Keyboard%20Interaction,-Enter) does not mention typeahead. The [example Accordion](https://www.w3.org/WAI/ARIA/apg/patterns/accordion/examples/accordion/) has inputs with typing allowed.


Before:

![Screenshot 2024-01-25 at 1 52 35 PM](https://github.com/adobe/react-spectrum/assets/15165138/db2a0fbd-6f43-46f2-8b43-bab1e2168069)

https://github.com/adobe/react-spectrum/assets/15165138/aec23968-fc5a-43dc-84f9-7b1bf0ed80ed

After:

![Screenshot 2024-01-25 at 1 52 52 PM](https://github.com/adobe/react-spectrum/assets/15165138/66802ef7-3ab6-4860-ba3f-01a64d353753)

https://github.com/adobe/react-spectrum/assets/15165138/a92170b7-d2e0-460b-b5d8-aab15a465aec





## ✅ Pull Request Checklist:


## 📝 Test Instructions:

1. `git checkout allow-typing-in-accordion`
2. `yarn jest Accordion`
3. `yarn start`
4. Visit [http://localhost:9003/?path=/story/accordion--with-input](http://localhost:9003/?path=/story/accordion--with-input)
5. Try typing inside the input
6. Observe spaces and other typing does not conflict with typeahead

## 🧢 Your Project:

Art of Problem Solving, internal tooling using `react-aria`
